README.md

```diff
@@ -14,6 +15,8 @@ The Amazon Chime React Component Library can be added to your web application co
 
 Documentation on these components and how to use them can be found on [Amazon Chime SDK React Component Library Documentation](https://aws.github.io/amazon-chime-sdk-component-library-react/).
 
+If you have some particular use cases, please go to [How-to guides](https://aws.github.io/amazon-chime-sdk-component-library-react/?path=/story/how-tos--page)
```

Comment:
May be we just say "Please check use-cases in our [How-to guides](https://aws.github.io/amazon-chime-sdk-component-library-react/?path=/story/how-tos--page) for more information" ?

Reply:
Sure, will update it